### PR TITLE
Docs: Move ESNext as default code example

### PR DESCRIPTION
**Pull request by mkaz**
## Description

Per [documentation guidelines](https://github.com/WordPress/gutenberg/blob/master/docs/contributors/document.md#code-examples), the ESNext examples should be listed first. This PR swaps the placement with ESNext and ES5 to do so.

## How has this been tested?

Confirm no code changes, tags are setup right. Hard to confirm without generating since the codetabs do not render in Github.

## Types of changes

Documentation.

### CODE REVIEW

File: docs/designers-developers/developers/internationalization.md [hunk 1]
```diff
@@ -34,58 +34,58 @@ function myguten_block_init() {
...
-import { registerBlockType } from '@wordpress/blocks';
+const { __ } = wp.i18n;
+const el = wp.element.createElement;
+const { registerBlockType } = wp.blocks;
 
 registerBlockType( 'myguten/simple', {
 	title: __( 'Simple Block', 'myguten' ),
 	category: 'widgets',
 
 	edit: () => {
```
Comment (nosolosw) [inline docs/designers-developers/developers/internationalization.md:76]:
```suggestion
	edit: function() {
```

File: docs/designers-developers/developers/internationalization.md [hunk 2]
```diff
@@ -34,58 +34,58 @@ function myguten_block_init() {
...
-				{ __( 'Hello World', 'myguten' ) }
-			</p>
+		return el(
+			'p',
+			{ style: { color: 'red' } },
+			__( 'Hello World', 'myguten' )
 		);
 	},
 
 	save: () => {
```
Comment (nosolosw) [inline docs/designers-developers/developers/internationalization.md:84]:
```suggestion
	save: function() {
```

**Review by oandregal — APPROVED** [review]
I've left two fixes (things that were already misspelled, but I guess fixing them in this PR is fine), other than that :ship:

**Comment by mkaz** [comment]
Thanks @nosolosw :+1:

**Pull request merged**